frontend/src/js/components/settings/CardSection.tsx

```diff
@@ -105,16 +107,16 @@ export const CardSection = ({ isSignUp, onClose, organization, onSubmit, onCardC
         <img src={stripeImage} />
       </div>
 
-      {isSignUp && <InfoText>Billing will be scheduled monthly, starting from today. You can cancel at any time.</InfoText>}
-
+      {isSignUp && <InfoText>{infoText ? infoText : 'Billing will be scheduled monthly, starting from today. You can cancel at any time.'}</InfoText>}
+      {summary}
       <div className="flexbox center-aligned margin-top-small">
         {!isSignUp && (
           <Button type="reset" disabled={loading} style={{ marginRight: 15 }}>
             Cancel
           </Button>
         )}
         <Button variant="contained" color="secondary" type="submit" disabled={errors || loading || empty || !isValid}>
-          {isSignUp ? 'Sign up' : 'Save'}
+          {isSignUp ? 'Confirm Subscription' : 'Save'}
         </Button>
       </div>
```

Comment:
to align with the design I would suggest 
```suggestion
<div className="flexbox center-aligned margin-top-small">
<Button type="reset" disabled={loading} style={{ marginRight: 15 }} onClick={onCancel}>
Cancel
</Button>
<Button variant="contained" color="secondary" type="submit" disabled={errors || loading || empty || !isValid}>
          {isSignUp ? 'Confirm Subscription' : 'Save'}
</Button>
</div>
```
as the components current uses will be phased out soon anyway + the missing click handler on the button wasn't raised so far 🙈